media/vscode.css

```diff
@@ -92,3 +92,8 @@ textarea::placeholder {
 table {
   border: var(--default-border-size) solid var(--border);
 }
+
+a{
+  background-color: var(--vscode-editor-background);
+  color: var(--vscode-textLink-foreground);
+}
```

Comment:
Hey! Thanks for your PR.

Doesn't this change only affect anchor tags? Did adding these lines of code change the colour for select options?

Nothing big but a spacing after `a` would be nice! 

#### Current Change 
```
a{
  background-color: var(--vscode-editor-background);
  color: var(--vscode-textLink-foreground);
}
```

#### Preferred way

```
a {
  background-color: var(--vscode-editor-background);
  color: var(--vscode-textLink-foreground);
}
```

Just to keep the code style throughout this project consistent.

Reply:
Yeah the options were fine they changed based on my vs code theme

Reply:
i was surprised as well but i can test it again just to make sure

Reply:
I just checked on my vscode with your recent change but it looks a bit odd on my side.

#### Before Change
<img width="350" alt="Select Option Menu" src="https://user-images.githubusercontent.com/83770081/192383205-3d12dc83-8972-4f83-93fd-154fbcc02ea8.png">

#### After Change
<img width="350" alt="Select Option Menu" src="https://user-images.githubusercontent.com/83770081/192383156-f2fe3f94-0401-4e3e-941b-43b11808c74d.png">

Reply:
Also, I see anchor tag CSS already declared on line 33, I don't think it's necessary to re-declare anchor tag CSS at the bottom of the stylesheet.

#### Stylesheet

<img width="350" alt="Stylesheet" src="https://user-images.githubusercontent.com/83770081/192383641-343d3d2c-e8a2-4469-be0c-b446fae2fc7d.png">

Reply:
the change was to change text colour and background based of va code theme, what would you like to stay the same cause i can modify

Reply:
Could there be a way to change text colour and background based on vscode theme but have the original style?

Reply:
could you elaborate when you mean original style?

Reply:
Original style as in the way it was originally implemented but handle cases such as different themes

#### Origianl Style
<img width="344" alt="Screen Shot 2022-09-27 at 6 32 17 AM" src="https://user-images.githubusercontent.com/83770081/192384554-7a2c5f15-6807-4396-9322-e219997cb70d.png">

Reply:
yeah okay i'll look into it tomorrow

Reply:
Also I just realized below code didn't actually chang the color and cause the result as you would have expected.

```
a {
  background-color: var(--vscode-editor-background);
  color: var(--vscode-textLink-foreground);
}
```

Removing below code actually made your select options visible.

```
  background: transparent;
  color: rgba(255, 255, 255, 0.78);
```

Reply:
how so since the rgb is white colour and background transparent which was the reason it didn't work for me the first time

Reply:
oh i just understood what you meant

Reply:
I'm not sure if it will work but try below code.

```
  color: var(--default-text);
```

This should fix the problem you were having earlier.

Reply:
And I guess you could add some opacity to have the same style as before but make it work for other vscode themes.

#### Current style

```
  background: transparent;
  color: rgba(255, 255, 255, 0.78);
```

#### Improved style
```
  color: var(--default-text);
  opacity: 0.78
```

and for background css, you probably have to test it out. Not sure if transparent will work on other vscode themes.

Reply:
ok thanks for the guidance i will fix it tomorrow

Reply:
![image](https://user-images.githubusercontent.com/74113508/192521257-7e72fa54-82bc-4492-87b6-6de04718f25a.png)
![image](https://user-images.githubusercontent.com/74113508/192521420-4a2438bd-3818-4de5-999f-8c908671e203.png)
do these two look good?

Reply:
@Cxxshyy Looks really nice. Thanks for this!

Reply:
Closing this PR since you made a new pull request